styles.css

```diff
@@ -170,4 +396,199 @@ a:link {
     top: 13px;
     bottom: 0;
   }
+
+  #w-title {
+    font-size: 10vw;
+    padding: 5vw;
+    margin-bottom: 2vw;
+  }
+
+  #works {
+    padding: 5vw;
+  }
+
+  .separator {
+    width: 7vw;
+    border-radius: 10px;
+    height: 0.8vw;
+    display: flex;
+    background-color: #d92424;
```

Comment:
- [ ] on the design the color swatch included  #36B37E  you replaced it with  #d0d9d4, I suggest you stick to the Figma design, as it is important for a front end to create a design exactly as it is without making personal changes


| Your design | ![bad color 1](https://user-images.githubusercontent.com/102636489/227703370-16fc29ec-34f7-410a-b1a9-30bce6dabc76.png) | ![bad color 2](https://user-images.githubusercontent.com/102636489/227703423-2b6fc1c4-de94-4d07-87a8-130de2fd65cb.png) | ![bad color 3](https://user-images.githubusercontent.com/102636489/227703448-d07ab307-9e36-47e1-9203-9c66c86419fa.png) |
| --- | --- | --- | --- |
| Figma's design | ![good color1](https://user-images.githubusercontent.com/102636489/227703497-e5124d41-d9ac-498d-86d3-81c7bdc80919.png) | ![good color2](https://user-images.githubusercontent.com/102636489/227703540-5cae44e3-f4fa-4543-98c1-c5fa917278df.png) | ![good color3](https://user-images.githubusercontent.com/102636489/227703589-89a97783-1acd-49c1-b3c2-92ccc11b8c1b.png) |

Reply:
all color pallete altered under request